README.md

```diff
@@ -40,6 +40,10 @@ Settings used by Github Action  to build the website (see .[.github/workflows/de
 
     git clone --recurse-submodules git@github.com:PyLadiesBerlin/website.git
 
+Or:
+
+    git submodule update --init --recursive
```

Comment:
I think this is a complementary step if the clone of the submodules doesn't work. I think it is better to write the error you got and this as a solution.